src/main/java/io/fabric8/maven/docker/config/LogConfiguration.java

```diff
@@ -58,6 +58,25 @@ public Boolean isEnabled() {
         return enabled;
     }
 
+    /**
+     * If explicitly enabled, or configured in any way and NOT explicitly disabled, return true.
+     *
+     * @return
+     */
+    public boolean isActivated() {
+        return enabled == Boolean.TRUE ||
+                (enabled != Boolean.FALSE && !isBlank());
```

Comment:
shouldn't this be better

```java 
enabled == Boolean.TRUE || (enabled == null && !isBlank());
```

which might be easier to understand.

Reply:
They'd be the same yes. But imo it's clearer that we want to explicitly check for false; the thing we want is that if not explicitly disabled, then we let !isBlank() decide if it is "activated" or not.